### PR TITLE
Project structure initialization

### PR DESCRIPTION
**Pull request by dmsolr**
1. code style checker plugin.
2. license header checker by using skywalking-eyes.
3. project structure:

-skywalking-transporter-plugin[root] 
 |-fetcher-plugins (agent)
 |-reporeter-plugins (oap)

### CODE REVIEW

File: .github/workflows/ci-it.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: CI AND IT
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```
Comment (wu-sheng) [inline .github/workflows/ci-it.yaml]: I think it is main?

File: .github/workflows/e2e.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+name: E2E
+
+on:
+  pull_request:
+    paths:
+      - '**'
+      - '!**.md'
+  push:
+    branches:
+      - master
```
Comment (wu-sheng) [inline .github/workflows/e2e.yaml]: Same here. Please test CI on your fork repo.

File: .licenserc.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Apache Software Foundation
```
Comment (kezhenxu94) [inline .licenserc.yaml]: This should not be Apache Software Foundation, should be SkyAPM. Also run skywalking-eyes locally (or on your forked repo) to see the wrong license headers and fix them

File: checkstyle/CHECKSTYLE_HEAD [hunk 4]
```diff
@@ -0,0 +1,18 @@
+^<\?xml version="1\.0" encoding="UTF-8"\?>$
```
Comment (kezhenxu94) [inline checkstyle/CHECKSTYLE_HEAD]: We don't need this if you have skywalking-eye configured

File: .github/workflows/ci-it.yaml [hunk 5]
```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (kezhenxu94) [inline .github/workflows/ci-it.yaml]: I also meant the copyright owner in all of these existing license header need to be SkyAPM

**Comment by dmsolr** [comment]
@kezhenxu94 @wu-sheng it is 
ready. please recheck

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**